Bangazon-Tinkr/Controllers/RubbishController.cs

```diff
@@ -41,6 +45,21 @@ public IActionResult GetAllRubbish()
 
             return Ok(allRubbish);
         }
+        // api/Rubbish/2/createRubbish
+        [HttpPost("{userId}/createRubbish")]
+        public IActionResult CreateNewRubbish(int userId, Rubbish rubbishToAdd)
```

Comment:
If we remove from the route we can remove the userId from here also.  You would want to have the userId in the body of what is being sent to create the new piece of rubbish.